packages/mdc-drawer/_mixins.scss

```diff
@@ -123,3 +122,11 @@
     @include mdc-theme-prop(color, $value);
   }
 }
+
+@mixin mdc-drawer-subtext-ink-color($color, $opacity) {
+  $value: rgba(mdc-theme-prop-value($color), $opacity);
+
+  .mdc-drawer__subtext {
+    @include mdc-theme-prop(color, $value);
+  }
+}
```

Comment:
Discussed offline: merge this back together with subtitle, and rename subtitle-text-color to subtitle-ink-color

Also make sure there's a newline at the end of the file after reverting (lint caught it)